client/consensus/manual-seal/src/error.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
I guess the dates in all files now need an update

Reply:
all the new files at least :)